src/commands/infra/scaffold.ts

```diff
@@ -160,7 +173,7 @@ export const parseVariablesTf = (data: string) => {
         }
         fields[elt[0]] = value;
       } else {
-        fields[elt[0]] = null;
+        fields[elt[0]] = "";
```

Comment:
Why null vs quotes?

Reply:
Because HCL cannot process "null", whereas both JSON and HCL files can process empty strings.